src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -68,7 +68,7 @@ featurizers:
 
     configuration_params:
       - type: uint32
-        name: windowSize
+        name: maxWindowSize

```

Comment:
>name: maxWindowSize [](start = 8, length = 19)

The order of these parameters should be updated to match the order in the object's constructor.

Reply:
Fixed.